front/cypress/support/form-builder.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Add an item from the toolbox to the form builder using keyboard navigation approach
+ * This command replicates the complex keyboard navigation logic for adding form elements
+ *
+ * @param toolboxSelector - The selector for the toolbox item (e.g., '#toolbox_number')
+ * @example cy.addItemToFormBuilder('#toolbox_number')
+ */
+Cypress.Commands.add('addItemToFormBuilder', (toolboxSelector: string) => {
```

Comment:
The new command relies on keyboard navigation because the regular drag and drop was not working properly or reliably in cypress.